tmt/base.py

```diff
@@ -3900,7 +3902,7 @@ def _stop_running_guests(self, run: Run) -> bool:
     def guests(self, run_ids: tuple[str, ...]) -> bool:
         """ Clean guests of runs """
         self.info('guests', color='blue')
-        root_path = Path(self.opt('workdir-root'))
+        root_path = Path(self.workdir_root)
```

Comment:
`Path()` may not be needed because `self.workdir_root` is already returning `Path` instances.

Reply:
And since this variable is used exactly once, on like 3913, we can easily not create it and use `self.workdir_root` when calling `generate_runs()`.

Reply:
Yeah, but  after this update, root_path will be called twice, we'd better create it,right?:)

Reply:
Yes, if you start logging it and therefore add a second use of that value, it looks tempting to store it in a local variable. I for one would still use `self.workdir_root`, because it's short, it doesn't harm readability, and brings no performance penalty :)

Reply:
This does not seem to be addressed yet.